framework/contracts/account/account/Cargo.toml

```diff
@@ -34,11 +32,15 @@ abstract-sdk = { workspace = true }
 abstract-std = { workspace = true }
 semver = { workspace = true }
 abstract-macros = { workspace = true }
-workspace-hack = { version = "0.1", path = "../../../workspace-hack" }
+cw721 = { version = "0.18.0" }
+cw721-base = { version = "0.18.0", features = ["library"] }
```

Comment:
Use/add types from `framework/packages/abstract-std/src/objects/ownership/cw721.rs` instead, or put it only in dev-deps if you need it for testing
```suggestion
```